CMakeLists.txt

```diff
@@ -124,7 +124,8 @@ endif()
 # ==============================================================================
 option(CUDAQ_BUILD_TESTS "Build cudaq tests" ON)
 option(CUDAQ_ENABLE_RPC_LOGGING "Enable verbose printout for client/server qpud connection." OFF)
-option(CUDAQ_TEST_MOCK_SERVERS "Enable Remote QPU Tests via Mock Servers." OFF)
+option(CUDAQ_TEST_MOCK_SERVERS "Enable Remote QPU Tests via Mock Servers." ON)
+option(CUDAQ_INTEGRATION_TEST_BRAKET "Run integration tests on Braket backend." ON)
```

Comment:
Why the separate variable specifically for the braket tests here?

Reply:
This isn't needed, will remove it.